lib/nrf_modem_lib/Kconfig

```diff
@@ -56,6 +56,82 @@ config NRF_MODEM_LIB_SENDMSG_BUF_SIZE
 	  the repacked message would not fit into the buffer, `sendmsg` sends
 	  each message part separately.
 
+comment "Heap and buffers"
+
+config NRF_MODEM_LIB_HEAP_SIZE
+	int "Library heap size"
+	default 512
+	range 512 2048
+	help
+	  Size of the heap buffer used by the library.
+	  This heap is allocated in the application's RAM.
+
+config NRF_MODEM_LIB_SHMEM_CTRL_SIZE
+	hex
+	default NRF_MODEM_SHMEM_CTRL_SIZE
+	help
+	  Size of the shared memory area used for control structures.
+	  This is a constant for a given library build, and is exported
+	  by the library via NRF_MODEM_SHMEM_CTRL_SIZE.
+
+config NRF_MODEM_LIB_SHMEM_TX_SIZE
+	int "TX region size"
+	range 1024 16384
+	default 4096
+	help
+	  Size of the shared memory area owned by the application.
+	  This area holds all outgoing data from the application, e.g. buffers passed to `send()`.
+	  The size of this buffer affects directly the largest payload that can sent be on AT sockets.
+
+config NRF_MODEM_LIB_SHMEM_RX_SIZE
+	int "RX region size"
+	range 1544 16384
+	default 4096
```

Comment:
I think this default should be larger, this buffer also include the messages and the lists.

Reply:
ok, upped to 8k